src/main.c

```diff
@@ -226,4 +235,12 @@ int main(int argc, char** argv) {
     }
 
     return EXIT_SUCCESS;
+
+    goto not_experimental_err; // silences a warning
+not_experimental_err:
+    fprintf(
+        stderr,
+        "Experimental option '-%c' passed in a non-EXPERIMENTAL build. Rebuild with 'make EXPERIMENTAL=1' to enable.\n",
+        opt);
+    return EXIT_FAILURE;
```

Comment:
Could wrap this in an ifndef too, but it'll surely get optimised out anyway.

Reply:
It does